app/src/main/AndroidManifest.xml

```diff
@@ -29,6 +29,10 @@
     <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES"
         tools:ignore="QueryAllPackagesPermission" />
 
+    <!-- Allows changing locales -->
+    <uses-permission android:name="android.permission.CHANGE_CONFIGURATION"
+        tools:ignore="ProtectedPermissions" />
```

Comment:
The warning that we're suppressing here has an important one: "Permission is only granted to system apps"

We definitely can't submit fenix to the Play store if we add this to any of our release variants.

Since we run our UI tests on debug builds IIRC. Can we add this permission to the app/src/debug/AndroidManifest.xml so that it's limited to only that variant as mentioned in the comments above (assuming we have no options and need to add this permission).

Reply:
Sure @jonalmeida! Thank you, that's what I wanted to know.